external/example-test/playlist.xml

```diff
@@ -17,7 +17,7 @@
 		TestKitGen can produce the equivalent make command lines to be executed in the build 
 	-->
 	<test>
-		<testCaseName>example-test</testCaseName>
+		<testCaseName>example_test</testCaseName>
```

Comment:
example-test directory will be removed https://github.com/adoptium/aqa-tests/issues/3181